### DISCUSSION TITLE
Use Markdown Version of Github Pages Without removing HTML file

### DISCUSSION THREAD

**Discussion by christopherkwok** [community/community#22200] [Other Feature Feedback, Questions, & Ideas]
<p>Sorry, I’m rather new to this, so any help or advice would be greatly appreciated on this topic!</p>
<p>I recently added an HTML file to my personal github pages repository so that I could track my progress with it over time. However, the HTML code isn’t quite ready to be displayed on my website yet and I was wondering if there was still a way to use the previous Markdown version of my Github website without removing the HTML file from my repository.</p>
<p>Thanks in advance!</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/christopherkwok">@christopherkwok</a>, welcome to the GitHub Support Community! If your files both have the same name (excluding the file extension) Pages will always choose the HTML version over the markdown version.</p>
<p>The best way to keep this file excluded from your site until it’s ready would be to store it in a separate branch, work on the file in that branch, and then merge that branch with the branch that hosts your Pages site when you’re ready to launch it.</p>

**Comment by christopherkwok** [comment]
<p>Ohhh perfect, thanks so much for the help!</p>
